backend/src/Wasm/DarkEditor.fs

```diff
@@ -55,13 +55,44 @@ let getStateForEval
     executingFnName = None }
 
 
+
+/// Any 'loose' exprs in the source are mapped without context of previous/later exprs
+/// so, a binding set in one 'let' will be unavailable in the next expr if evaluated one by one.
```

Comment:
I should either explain this better, or figure out another solution - I got stuck on this for a while so went with the first solution I could come up with.